README

```diff
@@ -78,7 +78,8 @@ debug::
 Enables debug output
 
 debug_file::
-Filename to write debug to, file must exist and be a regular file, or may be one of "stderr", "stdout", or "syslog". STDERR is the default.
+Filename to write debugging messages to. If this file is missing, nothing will be logged! This regular file **has to be created by you** 
+or **must exist and be a regular file** for anything getting logged to it. It is not created by pam-u2f on purpose (for security 
```

Comment:
Is there a missing piece at the end? The parenthesis is not closed.
Also, I would remove the exclamation point and replace `you` with `the user`.

Finally, would you mind adding this text to the man page as well? It's in `man/pam_u2f.8.txt`. Thanks!